src/shared/components/palette/case-flag/read-case-flag-field.component.html

```diff
@@ -4,4 +4,4 @@ <h3 class="govuk-heading-m">Party level flags</h3>
   <ccd-case-flag-table [flagData]="flagData"></ccd-case-flag-table>
 </div>
 <h3 class="govuk-heading-m">Case level flags</h3>
-<ccd-case-flag-table [flagData]="caseLevelCaseFlagData"></ccd-case-flag-table>
+<!-- <ccd-case-flag-table [flagData]="caseLevelCaseFlagData"></ccd-case-flag-table> -->
```

Comment:
@Daniel-Lam can you remove this commented code

Reply:
@RiteshHMCTS - this will be re-enabled at some point in the future, when we have case-level Case Flags data, hence left as a placeholder.

Reply:
then its fine @Daniel-Lam